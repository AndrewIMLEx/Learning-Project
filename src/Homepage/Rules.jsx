import React from "react";

class Rules extends React.Component{
    render() {
      return (
      <>
      <div className="rules">
        <h3>How to Play</h3>
        <ol>
            <li>Choose Difficulty Level-Easy(1-10) or Hard(1-100).</li>
            <li>The Computer will generate a random number between 1-10 for standard and 1-100 for Difficult.</li>
            <li>You only have 3 chances to guess a number and beat the Computer</li>
            <li>Don't worry the computer will notify you if your guess is higher or lower than the random generated number</li>
        </ol>
        </div>
      </>
      )
    }
}

export default Rules
