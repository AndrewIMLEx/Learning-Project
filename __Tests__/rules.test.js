import React from 'react';
import Rules from '../src/Homepage/Rules';
import renderer from 'react-test-renderer';
describe("Rules page",() =>{
    it('renders the rules page', () => {
      const tree = renderer.create(<Rules />).toJSON();
      expect(tree).toMatchSnapshot();
    })
  })
