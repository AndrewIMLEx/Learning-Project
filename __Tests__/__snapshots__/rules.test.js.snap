// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rules page renders the rules page 1`] = `
<div
  className="rules"
>
  <h4>
    How to Play
  </h4>
  <ol>
    <li>
      Choose Difficulty Level-Easy(1-10) or Hard(1-100).
    </li>
    <li>
      The Computer will generate a random number between 1-10 for standard and 1-100 for Difficult.
    </li>
    <li>
      You only have 
      <a
        style={
          {
            "color": "red",
          }
        }
      >
        3 chances
      </a>
       to guess a number and beat the Computer.
    </li>
    <li>
      Don't worry the computer will notify you if your guess is higher or lower than the random generated number.
    </li>
  </ol>
</div>
`;
